hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -408,6 +408,15 @@ public Pipeline createReplicationPipeline(HddsProtos.ReplicationType type,
     return scm.getPipelineManager().getPipelines();
   }
 
+  @Override
+  public Pipeline getPipeline(HddsProtos.PipelineID pipelineID)
+      throws IOException {
+    AUDIT.logReadSuccess(
```

Comment:
This might not be appropriate for Audit since it is not client access, we should log it in debug mode IMO.

Reply:
Done